the-faseb-journal.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0"
+    default-locale="en-US"
+    class="in-text" 
+    page-range-format="expanded"
+    demote-non-dropping-particle="sort-only"
+    initialize-with=". "
```

Comment:
@adam3smith I removed lines 7 and 8 (`initialize-with` and `name-as-sort-order` attributes), but this causes the names to be spelled out as 'first name [initial] last name'. The journal wants 'last name comma initials'. Is there a better way to achieve this than putting back lines 7 and 8?